lib/simplifyExpression/collectAndCombineSearch/LikeTermCollector.js

```diff
@@ -220,6 +223,19 @@ function getTermsForCollectingMultiplication(node) {
 }
 
 // A helper function for getTermsForCollectingMultiplication
+// e.g. nthRoot(x, 2), append '2': nthRootNode to terms dictionary
```

Comment:
The 2 is for the index, so I think naming it `nthRoot2` would be misleading. I think just having the `2` there would suffice? Not sure about the naming schema or this codebase.